Index: i3-4.1.1/man/i3-sensible-terminal.man
===================================================================
--- i3-4.1.1.orig/man/i3-sensible-terminal.man	2011-12-28 23:56:55.487581000 +0100
+++ i3-4.1.1/man/i3-sensible-terminal.man	2011-12-28 23:57:06.725802633 +0100
@@ -22,6 +22,7 @@
 It tries to start one of the following (in that order):
 
 * $TERMINAL (this is a non-standard variable)
+* x-terminal-emulator
 * urxvt
 * rxvt
 * terminator
