Description: i3-sensible-terminal: try x-terminal-emulator first
Author: Michael Stapelberg <stapelberg@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2012-09-19

---

Index: i3-4.3/i3-sensible-terminal
===================================================================
--- i3-4.3.orig/i3-sensible-terminal	2012-09-19 18:08:09.000000000 +0200
+++ i3-4.3/i3-sensible-terminal	2012-09-19 18:32:06.393883488 +0200
@@ -4,11 +4,7 @@
 #
 # This script tries to exec a terminal emulator by trying some known terminal
 # emulators.
-#
-# Distributions/packagers should enhance this script with a
-# distribution-specific mechanism to find the preferred terminal emulator. On
-# Debian, there is the x-terminal-emulator symlink for example.
-for terminal in $TERMINAL urxvt rxvt terminator Eterm aterm xterm gnome-terminal roxterm xfce4-terminal; do
+for terminal in $TERMINAL x-terminal-emulator urxvt rxvt terminator Eterm aterm xterm gnome-terminal roxterm xfce4-terminal; do
     if which $terminal > /dev/null 2>&1; then
         exec $terminal "$@"
     fi
